api/mapping/templates/mapping/scanreportvalue_list.html

```diff
@@ -41,12 +41,25 @@ <h2>Values</h2>
                             {{ object.frequency }}
                         </td>
                         <td>
-                            {% for concept in object.concepts.all %}
-                                <span>
-                                    {{ concept.concept.concept_id }} -
-                                    {{ concept.concept.concept_name }} ({{ concept.concept.domain_id }})
-                                </span> <a href="{% url 'scan_report_value_concept-delete' %}?scan_report_field_id={{ object.scan_report_field.id }}&scan_report_concept_id={{ concept.id }}">(delete)</a><br/>
-                            {% endfor %}
+			  <table class="table table-bordered table-sm">
```

Comment:
Is the code below assuming the user can provide a non valid, non standard concept? If so, that is not the case. When associating a new concept to a ScanReportConcept, the concept will be validated. Hence, the webapp does not need to keep track of what is the original concept.

Reply:
@py5gol this was marked as "a draft PR" so wasnt expecting or meaning for you to look at it yet.

Still, this is an important point, that would mean I could remove some unneeded features, and save me a lot of time with extra work... 
I started to play with adding this feature to save the source_concept_id as well as the concept_id because I thought it was needed, because of this example:

In Panther they mapped "4057568 - Black African (Race)" which is Non-Standard but valid (source_concept_id)
The standard relation is 38003600 - African (Race) (concept_id)

So if we/user was to enter `4057568` the validation should be rejecting it, so that we always have source_concept_id == concept_id?